app/pods/components/otp-enroll/component.js

```diff
@@ -25,10 +25,7 @@ export default class otpEnrollComponent extends Component {
       this.set('errorString', null)
       this.set('otpSent', true);
     })
-    .catch(err => {
-      console.error(err)
-      this.set('errorString', 'Cannot Send OTP to that email. Please use your registered email.')
-    })
+    .catch(err => displayError(err))
```

Comment:
No. Please read about [error handling](https://github.com/ember-cli/ember-ajax#error-handling) with `ember-ajax`. 

You need to read the response in case of error.